doc/migrations/3.1.1_3.1.1++.md

```diff
@@ -33,3 +33,30 @@ to:
 ]}.
 ```
 
+### RDBMS configuration migration
+
+RDBMS pools are no longer configured by a `{pool, rdbms, _}` tuple, instead using the generic outgoing pools mechanism.
+The connection configuration is now passed via `server` option of the pool insted of being configured via a top-level `{rdbms_server, _}` tuple.
```

Comment:
Should it be `odbc_server`? We are describing migration from 3.1.1 here, and as far as I remember there was `odbc_server` config option and not `rdbms_server`.